submissions/models.py

```diff
@@ -177,8 +179,18 @@ class Score(models.Model):
     """
     student_item = models.ForeignKey(StudentItem)
     submission = models.ForeignKey(Submission, null=True)
-    points_earned = models.PositiveIntegerField(default=0)
-    points_possible = models.PositiveIntegerField(default=0)
+    points_earned = models.DecimalField(
+        decimal_places=2,
+        default=Decimal('0.0'),
+        max_digits=6,
+        validators=[MinValueValidator(Decimal('0.0'))]
```

Comment:
Do users see an error message when one of these fails, or a 500 error?

Reply:
I guess consumers handle those errors, SGA does on the front end.

Reply:
Oh, didn't realize this doesn't have views/controllers.